docker/README.md

```diff
@@ -19,102 +19,107 @@ The haproxy listens on ports 5000 (connects to the primary) and 5001 (does load-
+    89e38ef02268   patroni         "/bin/sh /entrypoint…"   4 minutes ago    Up 4 minutes                                                                    demo-etcd1
+    998b2e0405bb   patroni         "/bin/sh /entrypoint…"   4 minutes ago    Up 4 minutes    0.0.0.0:5000-5001->5000-5001/tcp, :::5000-5001->5000-5001/tcp   demo-haproxy
+    17717a187445   patroni         "/bin/sh /entrypoint…"   4 minutes ago    Up 4 minutes                                                                    demo-patroni1
+    5f18c20896d5   patroni         "/bin/sh /entrypoint…"   4 minutes ago    Up 4 minutes                                                                    demo-etcd2
+    8d5677f6b16d   patroni         "/bin/sh /entrypoint…"   4 minutes ago    Up 4 minutes                                                                    demo-patroni3
+    051514b17f6a   patroni         "/bin/sh /entrypoint…"   4 minutes ago    Up 4 minutes                                                                    demo-patroni2
 
     $ docker logs demo-patroni1
-    2019-02-20 08:19:32,714 INFO: Failed to import patroni.dcs.consul
-    2019-02-20 08:19:32,737 INFO: Selected new etcd server http://etcd3:2379
-    2019-02-20 08:19:35,140 INFO: Lock owner: None; I am patroni1
-    2019-02-20 08:19:35,174 INFO: trying to bootstrap a new cluster
-    ...
-    2019-02-20 08:19:39,310 INFO: postmaster pid=37
-    2019-02-20 08:19:39.314 UTC [37] LOG:  listening on IPv4 address "0.0.0.0", port 5432
-    2019-02-20 08:19:39.321 UTC [37] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
-    2019-02-20 08:19:39.353 UTC [39] LOG:  database system was shut down at 2019-02-20 08:19:36 UTC
-    2019-02-20 08:19:39.354 UTC [40] FATAL:  the database system is starting up
+    2023-11-15 16:37:12,784 ERROR: Failed to get list of machines from http://etcd2:2379/v3beta: ValueError("invalid literal for int() with base 10: 'not_decided'")
+    2023-11-15 16:37:12,832 INFO: Selected new etcd server http://172.18.0.5:2379
+    2023-11-15 16:37:12,872 INFO: No PostgreSQL configuration items changed, nothing to reload.
+    2023-11-15 16:37:12,969 ERROR: watchprefix failed: ProtocolError("Connection broken: InvalidChunkLength(got length b'', 0 bytes read)", InvalidChunkLength(got length b'', 0 bytes read))
+    2023-11-15 16:37:12,924 INFO: Lock owner: None; I am patroni1
+    2023-11-15 16:37:13,015 INFO: failed to acquire initialize lock
+    2023-11-15 16:37:14,856 INFO: Lock owner: patroni2; I am patroni1
+    2023-11-15 16:37:14,856 INFO: trying to bootstrap from leader 'patroni2'
+    2023-11-15 16:37:14,984 INFO: Lock owner: patroni2; I am patroni1
+    2023-11-15 16:37:15,027 INFO: bootstrap from leader 'patroni2' in progress
+    2023-11-15 16:37:15,970 INFO: replica has been created using basebackup
+    2023-11-15 16:37:15,971 INFO: bootstrapped from leader 'patroni2'
```

Comment:
Previously it was showing logs of primary bootstrap on `patroni1`. In your case the bootstrap happened on `patroni2`. Either you need to show its logs, or maybe to a second attempt of `docker-compose up` so that bootstrap is executed on the first node.
Also, please pay an attention that plenty of log lines were skipped from the output (`...`).

Reply:
in logs, I didn't notice the bellow error when applying the `ETCDCTL_API=3` in `Dockerfile` : 

I'm not sure why this is happening , tried so many time `docker compose up -d` , but every time I get this error and its not bootingstrap on `patroni1`

```
Failed to get list of machines from http://etcd2:2379/v3beta: ValueError("invalid literal for int() with base 10: 'not_decided'")
```

Reply:
Thank You, did those and now the readme is updated and `patroni1` is leader and its bootstraping a new cluster on primary `patroni1` node